docs/CHANGELOG.asciidoc

```diff
@@ -31,6 +32,68 @@ This section summarizes the changes in each release.
 * <<release-notes-7.0.0-alpha1>>
 
 --
+
+
+[[release-notes-7.5.2]]
+== {kib} 7.5.2
+
+coming::[7.5.2]
+
+See <<breaking-changes-7.5, breaking changes in 7.5>>.
+
+[float]
+[[bug-7.5.2]]
+=== Bug fixes
+App architecture::
+* {kib} 7.0.0 URL field formatter doesn't render relative hyperlinks properly {pull}53789[#53789]
+* Fixes index pattern without timefield filter application {pull}54757[#54757]
+
+Console::
+* Handles double quote special case {pull}54474[#54474]
+
+Kibana app::
```

Comment:
This should be "Dashboard, Discover, Visualize" (listing all since these changes apply to multiple places and I think it's ok for a patch that we don't distinguish)